### PR TITLE
LPS-134283

### PR DESCRIPTION
**Pull request by mlan0**

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/pages/FormBuilder.es.js [hunk 1]
```diff
@@ -86,6 +86,18 @@ export const FormBuilder = () => {
 
 	const addToast = useToast();
 
+	useEffect(() => {
+		const sessionLength = Liferay.Session ? Liferay.Session.get('sessionLength') : 60000;
```
Comment (mlan0) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/pages/FormBuilder.es.js:90]: I did some tests and realized that every time I reload the form builder page, the Liferay.Session is null in the useEffect, that's why I made these changes compared to the implementation in the Page Editor.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/pages/FormBuilder.es.js [hunk 2]
```diff
@@ -86,6 +86,20 @@ export const FormBuilder = () => {
 
 	const addToast = useToast();
 
+	useEffect(() => {
+		const sessionLength = Liferay.Session
+			? Liferay.Session.get('sessionLength')
+			: 60000;
```
Comment (mlan0) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/pages/FormBuilder.es.js:92]: I did some tests and realized that every time I reload the form builder page, the Liferay.Session is null in the useEffect, that's why I made these changes compared to the implementation in the Page Editor.

**Comment by natocesarrego** [comment]
ci:forward

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**